packages/pigeon/lib/ast.dart

```diff
@@ -2,6 +2,8 @@
 // Use of this source code is governed by a BSD-style license that can be
 // found in the LICENSE file.
 
+// ignore_for_file: avoid_equals_and_hash_code_on_mutable_classes
```

Comment:
We shouldn't do this for a whole file; this is a useful warning, and we should have to explicitly override it each time.

Reply:
Do we even need it now?